buildpack-samples/README.md

```diff
@@ -0,0 +1,9 @@
+# Buildpack Samples
```

Comment:
@josephlewis42 Do we want to call these samples or "Buildpack Baselines". Per our discussion it seems like we might want these to be more than samples.

Reply:
I don't mind kicking the can down the road as far as naming them until we have a separate repo.

Reply:
Per the discussion in groups, do we want to even include these? (I say yes for now but probably not for long term).

Reply:
Well I don't think we have a place for them yet. So I think they need to stay in the repo for now.